v1-1/learn.md

```diff
@@ -7,7 +7,6 @@ permalink: /v1-1/learn/
 redirect_from:
   - /v1-1/learn/tools-ides
   - /v1-1/learn/tools-ides/
-  - /learn/
```

Comment:
please remove the same redirections in each page in learn in v1-1 dir

Reply:
Done